content/en/docs/tasks/administer-cluster/cgroupv1-maintenance-mode.md

```diff
@@ -0,0 +1,68 @@
+--- title: Moving cgroup v1 support into maintenance mode
+min-kubernetes-server-version: 1.31 content_type: task weight: 90 ---
+
+<!-- overview -->
+
+The Linux kernel community has made cgroup v2 the focus for new features,
+offering better functionality, a more consistent interface, and improved
+scalability. As a result, major Linux distributions and projects like systemd
+are phasing out support for cgroup v1. This trend creates an imperative for
+Kubernetes to align with these changes.
+
+
+<!-- steps -->
+
+
+## Why switch to cgroup v2?
```

Comment:
You may want to link something to what cgroups are before diving into the details. I'm sure there's content on the website already, but a quick link will give the audience the background needed quickly